source/runner/Executor.ts

```diff
@@ -40,10 +40,10 @@ export class Executor {
    * @param {string} file the filepath to the Dangerfile
    * @returns {void} It's a promise, so a void promise
    */
-
   async runDanger(file: string, runtime: DangerfileRuntimeEnv) {
     const results = await runDangerfileEnvironment(file, runtime)
-    await this.handleResults(results)
+    const signature = githubSignature(runtime.context.danger.github.pr)
```

Comment:
executionwise it doesn't make any changes, but feeling building signature also need to be handled in `handleResults`?